cloudformation/dashboard-data-extraction.yaml

```diff
@@ -0,0 +1,131 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  DashboardDataExtractFunction:
```

Comment:
Hmm, maybe you do need this as it failed the checkov step:
# checkov:skip=CKV_AWS_116: DLQ not needed for lambda